evap/evaluation/templates/contribution_formset.html

```diff
@@ -52,7 +52,7 @@
                 </td>
                 {% if editable %}
                     <td>
-                        {% if staff or not form.responsibility.value == "RESPONSIBLE" %}
+                        {% if staff or responsible %}
```

Comment:
@janno42 that if can simply be removed right? we always render the link, and the JS code removes it for the responsible as far as i understand